strings/lower.py

```diff
@@ -19,9 +17,7 @@ def lower(word: str) -> str:
     # converting to ascii value int value and checking to see if char is a capital
     # letter if it is a capital letter it is getting shift by 32 which makes it a lower
     # case letter
-    return "".join(
-        chr(ord(char) + 32) if 65 <= ord(char) <= 90 else char for char in word
-    )
+    return "".join(chr(ord(char) + 32) if "A" <= char <= "Z" else char for char in word)
```

Comment:
I am not sure but in my opinion this change causes a decrease of algorithm speed. Is comparing strings is not faster than comparing ints - in `ord()` case? @cclauss Your knowledge is need here! :)

Reply:
Anyone up for creating a timeit benchmark?